src/main/resources/assets/ebwizardry/texts/handbook_zh_cn.json

```
@@ -492,11 +501,26 @@
 				"#recipe wizard_leggings",
 				"#recipe wizard_boots",
 
-				"你可以用附魔台为巫师护具附魔,而因为魔法绸缎的固有属性，它们倾向于被附魔上能够使它们留存的更久的附魔。一些附魔，魔法，冰霜，雷电保护附魔，对巫师具有特殊的实用价值，这些附魔可以有效抵御其它巫师对你发起的进攻。",
+				"你可以用附魔台为巫师护具附魔,而因为魔法绸缎的固有属性，它们倾向于被附魔上能够使它们留存的更久的附魔。一些附魔，例如魔法，冰霜，雷电保护附魔，对巫师具有特殊的实用价值，这些附魔可以有效抵御其它巫师对你发起的进攻。",
 
 				"那些沉迷于练习单系魔法的巫师，有时会穿上可以给予额外魔法增益的护具。全套这样的护具在巫师群体中极为抢手，所以并不容易获得。",
 
-				"在传说中，最强大的巫师使用一种神秘的封印来提高他们护甲的能力。"
+				"当巫师们将基础魔法运用自如后，他们会选择某条道路，或者说，流派，来继续增进自身的技艺。他们有三种选择：贤者，战斗巫师以及巫术师。不同学派的成员会选择对原先的基础长袍进行改造，这样可以让长袍更为适应自身所需。因而不同流派的成员之间，所着长袍的样式各有其特点，他们唯一的相同之处便是：修改长袍往往需要一些来自异域的珍稀材料。",
+
+				"#image classes",
+
+				"贤者",
+
+				"穷其一生求知若渴，已使贤者在高效施法的技艺上臻于巅峰，他们能以最小量的魔力施放法术。贤者们所着丝绸长袍的华美掩盖了其强大的防护能力，要知道，贤者同样是娴熟的附魔者——只不过，这些长袍依然无法与坚固的金属板相匹敌。同时，长时间待在图书馆中固然安逸，但也让他们在战斗中显得无比迟缓。",
+
+				"战斗法师",
+
+				"战斗法师披铠待战，所着铠甲由技艺炉火纯青的锻造厂打造，内为坚韧布匹，外镶晶银镶板。在经历了刻苦的训练后，他们的速度与反应力都有了一定提升，因而他们能够更频繁地施放法术。然而，频繁施法的代价同样高昂，战斗法师在战斗中得小心，以免过快地耗尽魔力。",
+
+				"巫术士",
+
+				"巫术士为求索而生，他们不断推进那已知与未知之间的界限，探寻失落的奥术秘密。此外，他们还愿意去尝试那些被人们视若蛇蝎的禁忌法术，这种种原因使得他们能够突破传统训练的限制，以惊人的速度施放法术，并且减少魔力的使用量。他们的长袍十分轻盈，这对探险颇为有益，但与此同时，这种长袍的防御力在三种流派的巫师盔甲中是最低的。"
```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
				"巫术士为求索而生，他们不断推进那已知与未知之间的界限，探寻失落的奥术秘密。此外，他们还愿意去尝试那些被人们视若蛇蝎的禁忌法术，这种种原因使得他们能够突破传统训练的限制，以惊人的速度施放法术，并且减少魔力的使用量。他们的长袍十分轻盈，这对探险而言颇有益处，但与此同时，这种长袍的防御力在三个流派的巫师盔甲中是最低的。"
```